Install_WSL.rst

```diff
@@ -0,0 +1,106 @@
+.. -*- coding: utf-8; indent-tabs-mode:nil; -*-
+
+############################################################
+Spinal Cord Toolbox for Windows : Windows subsyetm for linux
+############################################################
+
+This documentation shows how to install the latest version of SCT. If you want install a different version of SCT please specify the version number you want.
+
+.. contents::
+..
+    1  Windows subsystem for linux 
+    2  environment preparation
+    3  SCT installation 
+    4  FSL installation 
+    5  Usage 
+
+Installation
+************
+
+
+#. Install `Xming <https://sourceforge.net/projects/xming/files/Xming/6.9.0.31/>`_.
+
+#. Install  `Windows subsystem for linux and initialize it <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`_. 
+
+	- Please install the Ubuntu 18.04 LTS distro. 
+
+Environment preparation
+***********************
+
+Run the following command to install various packages that will be needed to install FSL and SCT. This will require your password
+
+.. code-block:: sh
+
+    sudo apt-get -y update
+    sudo apt-get -y install gcc
+    sudo apt-get -y install unzip
+    sudo apt-get install -y python-pip python
+    sudo apt-get install -y psmisc net-tools
+    sudo apt-get install -y git
+    sudo apt-get install -y gfortran
+    sudo apt-get install -y libjpeg-dev
+    echo 'export DISPLAY=127.0.0.1:0.0' >> .profile
+
+
+Install SCT
+*********** 
+
+run the following command to download SCT:
+
+.. code:: sh
+
+    wget https://github.com/neuropoly/spinalcordtoolbox/archive/4.2.1.zip
```

Comment:
we should make the documentation version-agnostic. I anticipate many users will install 4.2.1 instead of the latest. Moreover, why not using `git` instead of `wget`? I will suggest some changes